src/Microsoft.Health.SqlServer.Api/Controllers/SchemaController.cs

```diff
@@ -9,30 +9,35 @@
 using EnsureThat;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.AspNetCore.Mvc.Routing;
 using Microsoft.Extensions.Logging;
-using Microsoft.Health.Fhir.Core.Features.Routing;
-using Microsoft.Health.Fhir.SqlServer.Api.Features.Filters;
-using Microsoft.Health.Fhir.SqlServer.Api.Features.Routing;
-using Microsoft.Health.Fhir.SqlServer.Features.Schema;
+using Microsoft.Health.SqlServer.Api.Features.Filters;
+using Microsoft.Health.SqlServer.Api.Features.Routing;
+using Microsoft.Health.SqlServer.Features.Schema;
 
-namespace Microsoft.Health.Fhir.SqlServer.Api.Controllers
+namespace Microsoft.Health.SqlServer.Api.Controllers
 {
     [HttpExceptionFilter]
     [Route(KnownRoutes.SchemaRoot)]
     public class SchemaController : Controller
     {
-        private readonly SchemaInformation _schemaInformation;
-        private readonly IUrlResolver _urlResolver;
+        private readonly ISchemaInformation _schemaInformation;
+        private readonly IScriptProvider _scriptProvider;
+        private readonly IUrlHelper _urlHelper;
         private readonly ILogger<SchemaController> _logger;
 
-        public SchemaController(SchemaInformation schemaInformation, IUrlResolver urlResolver, ILogger<SchemaController> logger)
+        public SchemaController(ISchemaInformation schemaInformation, IScriptProvider scriptProvider, IUrlHelperFactory urlHelperFactory, IActionContextAccessor actionContextAccessor, ILogger<SchemaController> logger)
         {
             EnsureArg.IsNotNull(schemaInformation, nameof(schemaInformation));
-            EnsureArg.IsNotNull(urlResolver, nameof(urlResolver));
+            EnsureArg.IsNotNull(scriptProvider, nameof(scriptProvider));
+            EnsureArg.IsNotNull(urlHelperFactory, nameof(urlHelperFactory));
+            EnsureArg.IsNotNull(actionContextAccessor, nameof(actionContextAccessor));
             EnsureArg.IsNotNull(logger, nameof(logger));
 
             _schemaInformation = schemaInformation;
-            _urlResolver = urlResolver;
+            _scriptProvider = scriptProvider;
+            _urlHelper = urlHelperFactory.GetUrlHelper(actionContextAccessor.ActionContext);
```

Comment:
Looks like my previous comment got lost. Would using the 'ControllerContext' property work here? It derives from `ActionContext`.